src/typeahead/index.tsx

```diff
@@ -418,7 +431,28 @@ export const Typeahead = factory(function Typeahead({
 								}}
 								valid={valid}
 							>
-								{{ label, leading }}
+								{{
+									label,
+									leading,
+									trailing: showDropDownButton && (
+										<button
```

Comment:
We couldn't use the dojo button here?

Reply:
This is following the same pattern that the `Select` widget is using. If we want to change that pattern both should be changed together.